src/Core/Components/Button/FluentToggleButton.razor.cs

```diff
@@ -0,0 +1,34 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// The FluentToggleButton component allows users to commit a change or trigger a toggle action via a single click or tap and
+/// is often found inside forms, dialogs, drawers (panels) and/or pages.
+/// </summary>
+public partial class FluentToggleButton : FluentButton
+{
+    /// <summary>
+    /// Gets or sets the mixed state of the component.
+    /// </summary>
+    [Parameter]
+    public bool Mixed { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets the pressed state of the button.
+    /// </summary>
+    [Parameter]
+    public bool Pressed { get; set; }
+
+    /// <summary />
+    protected new async Task OnClickHandlerAsync(MouseEventArgs e)
+    {
+        Pressed = !Pressed;
```

Comment:
You have a `pressedChanged` event for this web component. Maybe you could use it to be sure the `Pressed` value is always correct?

Reply:
As discussed. It uses ElementInternals which is something different than custom events. No changes necessary atm